tests/foreman/cli/test_remoteexecution.py

```diff
@@ -197,3 +235,17 @@ def test_positive_run_scheduled_job_template(self):
 
         @Assert: Verify the job was successfully ran after the designated time
         """
+        system_current_time = ssh.command('date +"%b %d %Y %I:%M%p"').stdout[0]
```

Comment:
That was my first try but I got into a timezone mismatch between the machine from which I ran Robotello and the server with Foreman, so this way I'm getting the server time. On a second thought, maybe it won't be a problem in automation, I'm keen to get educated here :)